### ISSUE TITLE
Checking Validation Loss

### ISSUE THREAD

**Issue by maesfahani** [cvlab-epfl/log-polar-descriptors#3]
Hi,

I am getting 0.083  Accuracy(FPR95) on the validation set for PTN, and 0.54 for STN; using the trained models you provided on the website. Is this result ok? 
I found that in the STN config file you set 
VAL_SPLIT:   'dl/patchesScale16_network_1.5px/PTN/val/'
should it be addressed to PTN for validating the model on the validation set?
